README.md

```diff
@@ -409,6 +409,34 @@ If you need to run a command after your tests have completed (such as removing c
 
 If you would prefer simply to clean up when Testem exits, you can use the `on_exit` option.
 
+Running browser code after tests complete
+-------------
+It is possible to send coverage reports or run other javascript in the browser by using the `afterTests` method. For asynchronous functions, just run a callback function as the first argument.
+
+```javascript
+Testem.afterTests(
+  //Asynchronously
+  function(cb) {
```

Comment:
I would prefer if we use the same API as you can already use in the config `function(config, data, callback)` to allow passing more additional details in the future.

https://github.com/testem/testem/blob/784e210f0619fcebd32f5ac51062396597a38e5f/examples/dynamic_config/testem.js#L16

I would also drop the `{"async": true}` and require the callback by default.